Changelog.md

```diff
@@ -1,11 +1,20 @@
 Drag and Drop XBlock changelog
 ==============================
 
+Version 4.0.0 (2023-01-31)
+--------------------------
+
+* Upgrade to be compatible with `bleach==6.0.0` which requires backwards
+  incompatible changes to code. We update our `install_requires` to make
+  this explicit.
+
+  BREAKING CHANGE: We can't run with bleach < 6.0.0
+
 Version 3.0.0 (2022-11-18)
 ---------------------------
 
-* Sanitize HTML tags to prevent XSS vulnerabilities.  
-  BREAKING CHANGE: Disallowed HTML tags (e.g. `<script>`) will no longer be rendered in LMS and Studio. 
+* Sanitize HTML tags to prevent XSS vulnerabilities.
+  BREAKING CHANGE: Disallowed HTML tags (e.g. `<script>`) will no longer be rendered in LMS and Studio.
```

Comment:
Nit: I suppose it was an automatic change from your editor, but double space means a line break in Markdown, so this changes the rendered result. If you'd like to get rid of it, please either change `  ` to ` \` (the result will be the same) or add a new line above `BREAKING CHANGE` (this will increase the space between these lines, but it's okay).

GitHub's rich diff does not handle it well, but you can check out how it is currently rendered [here](https://github.com/openedx/xblock-drag-and-drop-v2/blob/784e6ad5e4b09f2a0fd87d80d187bfe0ac36da7c/Changelog.md#version-300-2022-11-18).